### ISSUE TITLE
False positive for E502

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#68]
blah.py:

``` python
('\
    ' + ' \
')
```

```
$ ./pep8.py blah.py
blah.py:2:11: E502 the backslash is redundant between brackets
```
